File: add/codesnippet/CSharp/p-system.diagnostics.fil_16_1.cs
    private void GetPrivateBuild()
    {
        // Get the file version for the notepad.
        FileVersionInfo myFileVersionInfo =
            FileVersionInfo.GetVersionInfo(Environment.SystemDirectory + "\\Notepad.exe");

        // Print the private build number.
        textBox1.Text = "Private build number: " + myFileVersionInfo.PrivateBuild;
    }

File: add/codesnippet/CPP/p-system.diagnostics.fil_16_1.cpp
private:
    void GetPrivateBuild()
    {
        // Get the file version for the notepad.
        FileVersionInfo^ myFileVersionInfo =
            FileVersionInfo::GetVersionInfo( Environment::SystemDirectory + "\\Notepad.exe" );

        // Print the private build number.
        textBox1->Text = String::Concat( "Private build number: ", myFileVersionInfo->PrivateBuild );
    }